test/options.jl

```diff
@@ -49,7 +49,7 @@ MathProgBase.loadproblem!(m, -obj, A, rowub, [(:NonNeg,1:3)],[(:NonNeg,1:5)])
 MathProgBase.setwarmstart!(m, primal_sol; dual_sol = dual_sol, slack = slack)
 MathProgBase.optimize!(m)
 @test isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-12, rtol=0.0)
-@test !isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-14, rtol=0.0)
+@test !isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-13, rtol=0.0)
```

Comment:
I think you're going to have to make this waaaay looser. Like `atol=1e-6`.

Reply:
heh, It seems to me that I need to actually tighten this. all of tests fail with
```
Test Failed at /Users/runner/work/SCS.jl/SCS.jl/test/options.jl:52
  Expression: !(isapprox(MathProgBase.getobjval(m), -99.0, atol = 1.0e-13, rtol = 0.0))
```
i.e. solution is even more accurate ;)

Reply:
ok, I decided to comment those two lines as asking for `atol=1e-15` or smaller seems silly

Reply:
Urgh. I didn't see the `!` at the start of the line. That's a weird test. Why are we checking that we didn't find an accurate solution? Commenting seems fine.